adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepository.java

```diff
@@ -110,7 +110,7 @@
 
     // Map containing Shape Entities. A shape is a actually a collection of ShapePoint.
     // Entities are mapped on the values found in column shape_id  and shape_pt_sequence of GTFS file shapes.txt
-    private final Map<String, Map<Integer, ShapePoint>> shapePerIdShapePtSequence = new HashMap<>();
+    private final Map<String, SortedMap<Integer, ShapePoint>> shapePerIdShapePtSequence = new HashMap<>();
```

Comment:
This interface should have originally been `SortedMap` instead of `Map, because we use a `TreeMap` to order the contents by `shape_pt_sequence`. I had to change it to be able to use `SortedMap` in the utility implementation, because you can't cast from a `Map` to a `SortedMap`.